tests/unit/amo/pages/TestUserProfile.js

```diff
@@ -781,10 +787,13 @@ describe(__filename, () => {
     _setUserReviews({ store, userId: user.id });
 
     const dispatchSpy = sinon.spy(store, 'dispatch');
+    const errorHandler = createStubErrorHandler();
```

Comment:
As above, because you are using `neverCalledWithMatch`, you don't need to create and pass in an `errorHandler`.